dask/dataframe/io/parquet/arrow.py

```diff
@@ -1514,7 +1496,7 @@ def _make_part(
         """
 
         # Get full path (empty strings should be ignored)
-        full_path = fs.sep.join([p for p in [data_path, filename] if p != ""])
+        full_path = "/".join([p for p in [data_path, filename] if p != ""])
```

Comment:
@martindurant - One goal of this PR is to make it possible for the user to specify most (if not all) of the arguments needed to initialize a `pyarrow.dataset` or `fastparquet.ParquetFile` "dataset" object. For the pyarrow case, this can result in us passing around a pyarrow-based `fs` object.  We don't **need** to allow the user to pass in the `filesystem` option, but this feature now seems reasonably simple to support. The only sticky bit is that we would want to move away from using `fs.sep` throughout the code base. The pyarrow filesystem doesn't have this property, because it uses `"/"` in all cases (please correct me if I'm wrong @jorisvandenbossche).  Is this also true for fsspec? Can we just move to `"/"` throughout?

Reply:
Yes, you can use "/" everywhere when you have any fsspec instance. The only time that might now work, is if you `open()` a path directly, and that path is windows-style. If we are making fsspec instances, that will never happen.